src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -223,48 +225,60 @@ export default function TripPattern({
             </Typo.span>
           </div>
         </div>
+        <footer className={style.footer} onClick={() => setIsOpen(!isOpen)}>
+          <Button
+            title={
+              isOpen
+                ? t(PageText.Assistant.trip.tripPattern.seeLess)
+                : t(PageText.Assistant.trip.tripPattern.seeMore)
+            }
+            buttonProps={{
+              'aria-controls': 'assistant-details-accordion',
```

Comment:
Same comment as with ID previously. Generally I'd say that _all_ IDs in components should prefix with value from `useId`